ui/src/js/07-copy-code.js

```diff
@@ -8,7 +11,16 @@
   }
 
   function copyClick(e) {
-    const code = e.target.parentElement.innerText;
+    let code = e.target.parentElement.innerText;
+    const codeParentElement = e.target.parentElement;
+    const codeElement = codeParentElement.querySelector(codeElementSelector);
+    const codeLanguage = codeElement.dataset.lang;
+    const isShellLanguage = shellAliases.indexOf(codeLanguage) >= 0;
```

Comment:
```suggestion
    const isShellLanguage = shellAliases.includes(codeLanguage);
```

Reply:
I opened a follow up PR for babel + rollup -> https://github.com/OpenZeppelin/docs.openzeppelin.com/pull/298